test/connection-manager/index.spec.ts

```diff
@@ -5,12 +5,24 @@ import sinon from 'sinon'
 import { createNode } from '../utils/creators/peer.js'
 import { createBaseOptions } from '../utils/base-options.browser.js'
 import type { Libp2pNode } from '../../src/libp2p.js'
-import type { DefaultConnectionManager } from '../../src/connection-manager/index.js'
+import { DefaultConnectionManager } from '../../src/connection-manager/index.js'
 import { mockConnection, mockDuplex, mockMultiaddrConnection } from '@libp2p/interface-mocks'
 import { createEd25519PeerId } from '@libp2p/peer-id-factory'
 import { CustomEvent } from '@libp2p/interfaces/events'
 import { KEEP_ALIVE } from '@libp2p/interface-peer-store/tags'
 import pWaitFor from 'p-wait-for'
+import { multiaddr } from '@multiformats/multiaddr'
+import { Components } from '@libp2p/components'
+import { stubInterface } from 'ts-sinon'
+import type { Dialer } from '@libp2p/interface-connection-manager'
+import type { Connection } from '@libp2p/interface-connection'
+
+const defaultOptions = {
+  maxConnections: 10,
```

Comment:
should we relax the default value to 50 (or 100)? I imagine 10 is gonna annoy a lof of users unless they change.

Reply:
This is in a test?